http_examples.cpp

```diff
@@ -57,13 +57,16 @@ int main() {
 
             string name=pt.get<string>("firstName")+" "+pt.get<string>("lastName");
 
-            *response << "HTTP/1.1 200 OK\r\nContent-Length: " << name.length() << "\r\n\r\n" << name;
+            *response << "HTTP/1.1 200 OK\r\nContent-Length: " << name.length() << "\r\n" 
```

Comment:
You are right, it is just what is commonly used. I'll merge after I have understood the new headers later Today hopefully. Thank you again.